bundles/model/org.eclipse.smarthome.model.core/src/main/java/org/eclipse/smarthome/model/core/internal/folder/FolderObserver.java

```diff
@@ -236,7 +236,8 @@ private static void checkFile(final ModelRepository modelRepo, final File file,
             try {
                 synchronized (FolderObserver.class) {
                     if ((kind == ENTRY_CREATE || kind == ENTRY_MODIFY)) {
-                        if (parsers.contains(getExtension(file.getName()))) {
+                        // we omit parsing of hidden files possibly created by editors or operating systems
+                        if (parsers.contains(getExtension(file.getName())) && !file.getName().startsWith(".")) {
```

Comment:
This might not be the best approach.
Does this issue actually still exist? Please note https://github.com/eclipse/smarthome/pull/3069/files, https://github.com/eclipse/smarthome/pull/3112/files and https://github.com/eclipse/smarthome/pull/3148/files, which were about the same/similar issue.